src/main/java/com/valetparkingtracker/enterprise/controllers/ValetParkingTrackerController.java

```diff
@@ -33,15 +34,14 @@ public class ValetParkingTrackerController {
      * Handle the root (/) endpoint and return a start page.
      * @return
      */
-    @RequestMapping("/")
```

Comment:
Changed RequestMapping to Get/Postmapping for readability and used more updated Annotation